### ISSUE TITLE
Add a changelog

### ISSUE THREAD

**Issue by jletey** [nikitavoloboev/license-up#20]
We should definitely use https://github.com/yoshuawuyts/changelog

**Comment by jletey** [comment]
@nikitavoloboev Are we ready for a version 1.0.0 release ... there is still some open issues, but I feel like we have a core product down

Plus, the command line tool already says that we're on version 1.0.0

**Comment by nikivdev** [comment]
Agreed, let's release

**Comment by nikivdev** [comment]
I am not sure the process of releases is, I think pushing code to github is currently the way people do that. 

Then `go get ..` gets the latest version. We should just add a changelog and Git tag releases I think.

**Comment by jletey** [comment]
> We should just add a changelog and Git tag releases I think.

I agree ... the Git tag will probably have to come first (which I don't think I'll be able to do) https://github.com/nikitavoloboev/markdown-parser/issues/14#issuecomment-465583167

**Comment by nikivdev** [comment]
I made a tag & pushed it https://github.com/nikitavoloboev/license-up/releases

Not sure if this is what what was needed. I will read the rust changelog tool readme again.

**Comment by jletey** [comment]
> Not sure if this is what what was needed.

I'm still getting the same error

**Comment by nikivdev** [comment]
Ok will try solve

**Comment by jletey** [comment]
It turns out that you can't create a changelog without details in a `Cargo.toml` ... I've submitted a feature request (yoshuawuyts/changelog#25)

**Comment by jletey** [comment]
@nikitavoloboev Could you rename the latest release tag from `v1.0` to `1.0.0` ... this is what we have as the version in the cli, and plus this looks better when the changelog is generated

Do you want me to generate a changelog now (without pushing the `Cargo.toml`) or wait until the above feature request gets implemented?

**Comment by nikivdev** [comment]
Id wait until your issue gets addressed. In the meanwhile we can look for another changelog generator that is good?

**Comment by jletey** [comment]
Sounds good!

**Comment by jletey** [comment]
How about [Github Changelog Generator](https://github.com/github-changelog-generator/github-changelog-generator)? It looks pretty nice and we can have Travis run it every time we push something ... the only downside is that it doesn't include commits (only issues and PRs)

**Comment by nikivdev** [comment]
Not too fond of it. Lets wait for what the dev of rust changelog answers.

**Comment by jletey** [comment]
Will do
